src/Disks/ObjectStorages/FakeMetadataStorageFromDisk.h

```diff
@@ -82,7 +76,7 @@ class FakeMetadataStorageFromDiskTransaction final : public IMetadataTransaction
 
     const IMetadataStorage & getStorageForNonTransactionalReads() const final;
 
-    void commit() final;
+    void commit() final {}
```

Comment:
Do you understand why we need this `FakeMetadataStorageFromDisk`? It seems it's used if a disk is not remote, but if the disk is not remote (e.g. DiskLocal) it doesn't need a metadata storage at all...

Reply:
It's just fake wrapper for code migration.